crates/webidl/src/first_pass.rs

```diff
@@ -45,6 +45,7 @@ pub(crate) struct FirstPassRecord<'src> {
 pub(crate) struct InterfaceData<'src> {
     /// Whether only partial interfaces were encountered
     pub(crate) partial: bool,
+    pub(crate) has_no_interface: bool,
```

Comment:
the negative is a bit funky (vs `has_interface: bool`) but I guess it makes sense in this case

Reply:
Nah seems reasonable to me to switch!